fpmsyncd/fpmsyncd.cpp

```diff
@@ -122,17 +136,26 @@ int main(int argc, char **argv)
                  * select() loop.
                  * Note:  route reconciliation always succeeds, it will not be done twice.
                  */
-                if (temps == &warmStartTimer || temps == &eoiuHoldTimer)
+                if (temps == &warmStartTimer || temps == &eoiuHoldTimer || temps == &reconcileHoldTimer)
                 {
+                    bool readyToReconcile = false;                    
                     if (temps == &warmStartTimer)
                     {
+                        readyToReconcile = true;
                         SWSS_LOG_NOTICE("Warm-Restart timer expired.");
                     }
+                    else if (temps == &reconcileHoldTimer)
+                    {
+                        readyToReconcile = true;
+                        SWSS_LOG_NOTICE("Warm-Restart reconcile hold timer expired.");
+                    }
                     else
                     {
+                        readyToReconcile = sync.isReadyToReconcile();
```

Comment:
This will basically invalidate the eoiu design for fast reconciliation.  Understood we probably have to rely on orchagent reconciliation status, then the DEFAULT_ROUTING_RECON_CHECK_INTERVAL should be reduced to way smaller to take advantage of eoui.

Reply:
DEFAULT_ROUTING_RECON_CHECK_INTERVAL timer is only fallback timer. Is will only come into action when the replay/reconcile does not happen in given time. Hence it should not affect eoui handling.

Reply:
Got your point. When eoiuHoldTimer expires, that means control plane ( BGP ) has converged.  So now we just need to wait for orchangent reconcile and proceed once that is done. To implement this, will restart reconCheck timer with 2 seconds when eoiuHoldTimer expires, to check if orchagent has reconciled. Will implement this change.